src/content/guides/patron-not-present.mdoc

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Git is representing the changes as an insert / delete so I've called out the changes here
- File extension updated to `mdoc`
- We no longer need to explicitly import the custom fence component
- Code fences no longer need to be wrapped in a custom component
- Custom attributes like the code fence title need to be wrapped with markdoc tags